src/components/NetworkInfo/NetworkInfo.js

```diff
@@ -97,22 +97,6 @@ const TooltipContentRow = styled.div`
 	justify-content: space-between;
 `;
 
-const QuestionMarkIcon = styled.div`
-	cursor: pointer;
-	display: flex;
-	justify-content: center;
-	align-items: center;
-	border-radius: 50%;
-	width: 12px;
-	height: 12px;
-	background-color: ${(props) => props.theme.colors.accentL1};
-	margin-left: 4px;
-`;
-
-const QuestionMarkStyled = styled(QuestionMark)`
-	height: 8px;
-`;
-
```

Comment:
This question mark icon kept appearing in a lot of places, so I just moved it to shared.